content/docs/command-reference/init.md

```diff
@@ -73,7 +73,7 @@ might happen that Git repository root is located at different path than the DVC
 project root.)
 
 DVC project root defines the scope for the most DVC commands. Mostly meaning
-that all DVC-file under the root path are being analyzed.
+that all `.dvc` file under the root path are being analyzed.
```

Comment:
```suggestion
that all `.dvc` files and `dvc.yaml` under the root path are being analyzed.
```

Reply:
@shcheklein (correct Ivan)/ @efiop do you know if `dvc.yaml` can be in a subdirectory? Or additional `dvc.yaml` files in subdirs will be supported soon?

Reply:
@jorgeorpinel Yes, dvc.yaml's can be in any directory inside of the project. You can have multiple of them. Already supported.

Reply:
Ah! Thanks for the info Ruslan 🙂

Reply:
Moved this one to https://github.com/iterative/dvc.org/pull/1372#discussion_r435676484